AWS CLI - Idaptive V1/AWSCLI.py

```diff
@@ -31,7 +31,7 @@ def get_environment(args):
         tenant = tenant + ".centrify.com"
     name = tenant.split(".")[0]
     tenant = "https://" + tenant
-    cert = "cacerts_" + name + ".pem"
```

Comment:
keep this line but comment out.
Add additional comment to explain that customer can uncomment to configure Certificate Pinning.